src/snowflake/connector/connection.py

```diff
@@ -657,6 +662,10 @@ def connection_diag_log_path(self):
     def connection_diag_whitelist_path(self):
         return self._connection_diag_whitelist_path
 
+    @property
+    def connection_diag_allowlist_path(self):
+        return self._connection_diag_allowlist_path
```

Comment:
see my previous question about making it private.

also you can check this PR: https://github.com/snowflakedb/snowflake-connector-python/pull/1680/files
to see how we want to deprecate a public API

Reply:
See above.